ansible/roles/svc_traefik/tasks/setup_traefik.yml

```diff
@@ -0,0 +1,31 @@
+---
+# code: language=ansible
+# Path: ansible/roles/svc_traefik/tasks/setup_traefik.yml
+# This file contains the tasks to setup the Docker container for Traefik
+- name: Pull traefik image
+  community.docker.docker_image_pull:
+    name: docker.io/traefik
+    tag: latest
+    pull: always
+    timeout: 10
```

Comment:
Note: If we often set the timeout to 10s, let's have it in the env vars ([DOCKER_TIMEOUT](https://docs.ansible.com/ansible/latest/collections/community/docker/docsite/scenario_guide.html#envvar-DOCKER_TIMEOUT))

Reply:
https://deerhide.atlassian.net/browse/INFRA-111